src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -427,38 +455,71 @@ private CodeType CreateModelClasses(OpenApiUrlSpaceNode rootNode, OpenApiUrlSpac
             // one of object
             // any of object
 
+            
             if (originalReference == null)  // Inline schema, i.e. specific to the Operation
-            {//TODO
-                var codeClass = new CodeClass(codeNamespace) { Name = operation.OperationId + "Response" };
+            {
+                var namespaceName = GetNamespaceNameForModelByOperationId(operation.OperationId);
+                var ns = codeNamespace.GetRootNamespace().GetNamespace(namespaceName);
+                if(ns == null)
```

Comment:
embrace curly braces for single lines :)

Reply:
"we shall never surrender!" ;-)